fix(guide): Update Challenges Style Guide to recommend triple backticks instead of blockquote (#36069)
freeCodeCamp/freeCodeCamp: docs/style-guide-for-curriculum-challenges.md

- Language keywords go in `<code>` tags. For example, HTML tag names or CSS property names
- The first instance of a keyword when it's being defined, or general keywords (i.e. "object" or "immutable") go in `<dfn>` tags

> - Small pieces of code examples go in `<code>` tags. See example below:

Reviewer: I suggest a little more detail here.
```suggestion
- References to code parts (i.e. function, method or variable names) should be wrapped in `<code>` tags. See example below:
```

````

> <code>thisIsAnExampleFunction()</code>

Reviewer: The following is a typical use of code tags in the instructions section or a test's`text`.
```suggestion
Use <code>parseInt</code> to convert the variable <code>realNumber</code> into an integer.
```

````

> - Multi-line code examples go between two sequences of triple backticks. See example below:

Reviewer: ```suggestion
- Multi-line code blocks **must be preceded by an empty line**.  The next line must start with three backticks followed immediately by one of the [ supported languages](https://prismjs.com/#supported-languages).  To complete the code block, you must start a newline only has three backticks.

**Note:** If you are going to use an example code in YAML, use `yml` instead of `yaml`.

See example below:
```

Reviewer 2: The tests are parsed by looking for `yml` blocks and incorporating them.  This means it should be the other way around. 'yaml' is fine, 'yml' is not.

Reviewer: @ojeytonwilliams Oops!  I have updated the suggestion above.

````

> ```{language}

Reviewer: ````suggestion
The following is an example of code:

```{language}
````

[YOUR CODE HERE]

```
````

> The languages supported are listed [here](https://prismjs.com/#supported-languages). If you are going to use an example code in YAML, use `yml` instead of `yaml`.

Reviewer: ```suggestion
```

- Language keywords go in `<code>` tags. For example, HTML tag names or CSS property names
- The first instance of a keyword when it's being defined, or general keywords (i.e. "object" or "immutable") go in `<dfn>` tags

> - Single line code examples go in `<code>` tags

Reviewer: I think this line needs to be changed, because it is not that you have to use `<code>` tags for one line code segments.  It is for surrounding very short code (i.e. variable names, function/method names).  I think showing an example is best here.

> - Multi-line code examples go between two sequences of triple backticks. Example:

Reviewer: ```suggestion
- Multi-line code examples go between two sequences of triple backticks. See example below:
```

````
```{language}

[YOUR CODE HERE]

```
````
The languages supported are listed [here](https://prismjs.com/#supported-languages)

> - Multi-line code examples go in `<blockquote>` tags, and use the `<br>` tag to separate lines. For HTML examples, remember to use escape characters to represent the angle brackets

Reviewer: This line no longer needs to be in this document.
```suggestion
```
